benchmark/cxx-source/ReadAccessor.swift

```diff
@@ -0,0 +1,35 @@
+// Subscripts.swift - Very brief description
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// -----------------------------------------------------------------------------
+///
+/// This is a simple test that reads a non trivial C++ struct using an imported
+/// subscript thousands of times.
+///
+// -----------------------------------------------------------------------------
+
+import TestsUtils
+import CxxSubscripts
+
+public let benchmarks = [
+  BenchmarkInfo(
+    name: "ReadAccessor",
+    runFunction: run_ReadAccessor,
+    tags: [.validation, .bridging])
+]
+
+@inline(never)
+public func run_ReadAccessor(_ N: Int) {
+  var collection = NonTrivialArrayByRef()
+  for _ in 0...(N * 10_000) {
+    let local = collection[0].f;
```

Comment:
Depending on how you write this test, try using `i` here instead of `_`. This will probably get optimized away.